Makefile

```diff
@@ -18,8 +26,28 @@ FFMPEG_OPTS ?= --prefix=$(FFMPEG_BUILD) \
                --enable-parser=h264 \
                --enable-parser=hevc
 
+CGO_LDFLAGS := -L$(FFMPEG_PREFIX)/lib
+
+ifeq ($(TARGET_S),android)
+	export CGO_ENABLED = 1
+	NDK_ROOT ?= $(HOME)/Library/Android/Sdk/ndk/26.1.10909125
+	export CC = $(NDK_ROOT)/toolchains/llvm/prebuilt/$(UNAME_S)-x86_64/bin/aarch64-linux-android$(API_LEVEL)-clang
+	API_LEVEL ?= 30
+	FFMPEG_OPTS += --target-os=android \
+                   --arch=aarch64 \
+                   --cpu=armv8-a \
+                   --enable-cross-compile \
+                   --sysroot=$(NDK_ROOT)/toolchains/llvm/prebuilt/$(UNAME_S)-x86_64/sysroot \
+                   --cc=$(NDK_ROOT)/toolchains/llvm/prebuilt/$(UNAME_S)-x86_64/bin/aarch64-linux-android$(API_LEVEL)-clang \
+                   --cxx=$(NDK_ROOT)/toolchains/llvm/prebuilt/$(UNAME_S)-x86_64/bin/aarch64-linux-android$(API_LEVEL)-clang++
+	CGO_CFLAGS += -I$(NDK_ROOT)/toolchains/llvm/prebuilt/$(UNAME_S)-x86_64/sysroot/usr/include \
+                  -I$(NDK_ROOT)/toolchains/llvm/prebuilt/$(UNAME_S)-x86_64/sysroot/usr/include/aarch64-linux-android
+	CGO_LDFLAGS += -L$(NDK_ROOT)/toolchains/llvm/prebuilt/$(UNAME_S)-x86_64/sysroot/usr/lib
+	GO_TAGS ?= -tags no_cgo
+endif
+
 CGO_LDFLAGS := -L$(FFMPEG_BUILD)/lib
-ifeq ($(UNAME_S),Linux)
+ifeq ($(UNAME_S),linux)
 	CGO_LDFLAGS := "$(CGO_LDFLAGS) -l:libjpeg.a"
```

Comment:
Does this mean we are not going to link against `libjpeg` on android? If so, why not and will the module still work?